index.html

```diff
@@ -650,7 +746,7 @@ <h4>Automation via Workflows</h4>
     },
     "refreshService": {
       "type": "AutoRefresh2021",
-      "url": "https://example.edu/workflows/refresh-degree",
+      "url": "https://example.edu/workflows/refresh-degree/start",
```

Comment:
how much of `{name}` can be carried in the post body?

Reply:
There is a debate to be had over where it belongs. There are good arguments to put it in either place. I've raised this issue to track the debate: https://github.com/w3c-ccg/vc-api/issues/256